### PR TITLE
update UpdateIssue function on issues.go (#6)

### PR DESCRIPTION
**Pull request by antony360**
Added fields (now the structure is complete) on `UpdateIssue` function : 
- Confidential
- DueDate
- DiscussionLocked

### CODE REVIEW

File: issues.go [hunk 1]
```diff
@@ -278,13 +278,16 @@ func (s *IssuesService) CreateIssue(pid interface{}, opt *CreateIssueOptions, op
...
-	UpdatedAt   *time.Time `url:"updated_at,omitempty" json:"updated_at,omitempty"`
+	Title            *string    `url:"title,omitempty" json:"title,omitempty"`
+	Description      *string    `url:"description,omitempty" json:"description,omitempty"`
+	Confidential     *bool      `url:"confidential,omitempty" json:"confidential,omitempty"`
+	AssigneeID       *int       `url:"assignee_id,omitempty" json:"assignee_id,omitempty"`
+	MilestoneID      *int       `url:"milestone_id,omitempty" json:"milestone_id,omitempty"`
+	Labels           Labels     `url:"labels,comma,omitempty" json:"labels,omitempty"`
+	StateEvent       *string    `url:"state_event,omitempty" json:"state_event,omitempty"`
+	UpdatedAt        *time.Time `url:"updated_at,omitempty" json:"updated_at,omitempty"`
+	DueDate          *string    `url:"due_date,omitempty" json:"due_date,omitempty"`
```
Comment (svanharmelen) [inline issues.go]: `DueDate` should be of the `*ISODate`

**Comment by svanharmelen** [comment]
One small comment, looks good otherwise!

**Comment by antony360** [comment]
Right ! Done !

**Comment by svanharmelen** [comment]
Thanks @antony360! LGTM!

**Pull request merged**